src/terms/ewald.jl

```diff
@@ -34,7 +34,7 @@ function energy_ewald(model::Model{T}; kwargs...) where {T}
     # terms is equal and of opposite sign. See also the PSP correction term, where
     # n_electrons is used synonymously for sum of charges
     charges = T.(charge_ionic.(model.atoms))
-    @assert sum(charges) == model.n_electrons
+    isnothing(model.n_electrons) || @assert sum(charges) == model.n_electrons
```

Comment:
I find this tricky to understand... !isnothing(nel) && assert seems more readable? (same below)